docs/preparation/index.md

```diff
@@ -1,15 +1,71 @@
-# Introduction
+# Préparation
+
+Avant de commencer a preparer votre tournoi, vous devez definir vos objectifs, et trouver un perimetre. Est-ce que vous voulez organiser une avant-premiere semi-competitive a portee locale, ou un tournoi a 128 joueurs qualifiant pour le prochain LEC ?
```

Comment:
purée TO LEC CDF PAF faut vraiment que tu mettes un glossaire :D

Reply:
Il y a un moyen intelligent de le faire ? Un moyen ou je ne mets pas des liens a la main partout vers une page de glossaire ?

Le reve c'est que je definis un glossaire quelque part (une page ?) et qu'une *chose* met des tooltips la ou c'est pertinent :disguised_face:

Reply:
Apres 30 secondes a google : https://squidfunk.github.io/mkdocs-material/reference/tooltips/

Reply:
PAF c'est pour Poele A Frire. Comme dans "Comment va tu yau de PAF"

Reply:
J'ai rajoute un glossaire, forcement j'ai pu rate des termes, tu me diras au fil de l'eau si tu en vois :O